docs/modules/publishednodes_2.9.json

```diff
@@ -0,0 +1,124 @@
+// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
```

Comment:
Tet's stick to the 2.8 for product version for now. We're not sure about the versioning of the next release.